opendbc/car/honda/carstate.py

```diff
@@ -92,6 +92,8 @@ def __init__(self, CP):
       self.gearbox_msg = "GEARBOX_ALT"
     elif CP.transmissionType == TransmissionType.manual:
       self.gearbox_msg = "GEARBOX_ALT_2"
+    elif CP.carFingerprint == CAR.HONDA_CRV_6G:
+      self.gearbox_msg = "GEARBOX_15T"
```

Comment:
Move up with Accord check?

Reply:
We could, but then we would also need to add the check in interface to assign the CVT transmission type